_docs/reference/cli/namespace.md

```diff
@@ -7,41 +7,37 @@ module: reference/cli/namespace
 
 # Namespaces
 
-Namespaces help different projects or teams share a StorageOS cluster. No
-namespaces are created by default, and users can have any number of namespaces.
```

Comment:
Docker and K8s docs don't do this, so I'm disinclined to do so. If you feel strongly about this then I can do it.